plugins/volumeFilter_test.go

```diff
@@ -804,27 +829,52 @@ func TestVolumeFilterFn_QuoteCap_Ignore(t *testing.T) {
 	}
 	for _, k := range testCases {
 		// convert to common format accepted by runTestVolumeFilterFn
-		// doing this explicitly here is easier to read rather than if we were to add "logic" to convert it to a standard format
-		inputOp := makeSellOpAmtPrice(k.inputAmount, k.inputPrice)
-
-		var wantOp *txnbuild.ManageSellOffer
+		// test sell action
+		sellName := fmt.Sprintf("%s, sell", k.name)
+		inputSellOp := makeSellOpAmtPrice(k.inputAmount, k.inputPrice)
+		var wantSellOp *txnbuild.ManageSellOffer
 		if k.wantPrice != nil && k.wantAmount != nil {
-			wantOp = makeSellOpAmtPrice(*k.wantAmount, *k.wantPrice)
+			wantSellOp = makeSellOpAmtPrice(*k.wantAmount, *k.wantPrice)
 		}
 
 		runTestVolumeFilterFn(
 			t,
-			k.name,
+			sellName,
 			volumeFilterModeIgnore,
 			queries.DailyVolumeActionSell,
 			nil,                   // base cap nil because this test is for the QuoteCap
 			pointy.Float64(k.cap), // quote cap
-			nil,                   // baseOTB nil because this test is for the BaseCap
+			nil,                   // baseOTB nil because this test is for the QuoteCap
```

Comment:
looks like this comment was fixed to say QuoteCap instead of BaseCap. great catch!

It's sometimes easer to review these minor changes and not skip over them if they are highlighted in the PR with a comment, esp. since it breaks from the general pattern of the rest of the PR